ER_BAD_FIELD_ERROR: Unknown column 'belgeTipi' in 'field list'
->Error: ER_BAD_FIELD_ERROR: Unknown column 'belgeTipi' in 'field list'
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:107:40)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:461:44
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Tue Jun 18 2019 19:16:11 GMT+0300 (GTB Daylight Time)
ER_BAD_FIELD_ERROR: Unknown column 'belgeTipi' in 'field list'
->Error: ER_BAD_FIELD_ERROR: Unknown column 'belgeTipi' in 'field list'
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:107:40)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Tue Jun 18 2019 19:16:11 GMT+0300 (GTB Daylight Time)
ER_FIELD_SPECIFIED_TWICE: Column 'A' specified twice
->Error: ER_FIELD_SPECIFIED_TWICE: Column 'A' specified twice
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:107:40)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:462:44
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Tue Jun 18 2019 19:19:15 GMT+0300 (GTB Daylight Time)
ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '11-14-2019' for column 'belgeTarihi' at row 1
->Error: ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '11-14-2019' for column 'belgeTarihi' at row 1
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:107:40)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Tue Jun 18 2019 19:19:15 GMT+0300 (GTB Daylight Time)
Quit inactivity timeout
->Error: Quit inactivity timeout
    at Quit.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:160:17)
    at emitNone (events.js:106:13)
    at Quit.emit (events.js:208:7)
    at Quit._onTimeout (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:124:8)
    at Timer._onTimeout (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Timer.js:32:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.quit (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:91:23)
    at Connection.end (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:242:18)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:35:29)
    at new Promise (<anonymous>)
    at Database.close (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:34:16)
    at Query.connection.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:28:35)
    at Query.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at Query._callback (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:491:16)
    at Query.Sequence.end (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:83:24)
Tue Jun 18 2019 19:21:07 GMT+0300 (GTB Daylight Time)
ER_FIELD_SPECIFIED_TWICE: Column 'A' specified twice
->Error: ER_FIELD_SPECIFIED_TWICE: Column 'A' specified twice
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:107:40)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:462:44
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Tue Jun 18 2019 19:21:07 GMT+0300 (GTB Daylight Time)
ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '11-14-2019' for column 'belgeTarihi' at row 1
->Error: ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '11-14-2019' for column 'belgeTarihi' at row 1
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:107:40)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Tue Jun 18 2019 19:21:07 GMT+0300 (GTB Daylight Time)
Quit inactivity timeout
->Error: Quit inactivity timeout
    at Quit.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:160:17)
    at emitNone (events.js:106:13)
    at Quit.emit (events.js:208:7)
    at Quit._onTimeout (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:124:8)
    at Timer._onTimeout (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Timer.js:32:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.quit (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:91:23)
    at Connection.end (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:242:18)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:35:29)
    at new Promise (<anonymous>)
    at Database.close (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:34:16)
    at Query.connection.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:28:35)
    at Query.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at Query._callback (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:491:16)
    at Query.Sequence.end (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:83:24)
Tue Jun 18 2019 19:21:57 GMT+0300 (GTB Daylight Time)
ER_FIELD_SPECIFIED_TWICE: Column 'A' specified twice
->Error: ER_FIELD_SPECIFIED_TWICE: Column 'A' specified twice
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:107:40)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:462:44
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Tue Jun 18 2019 19:21:57 GMT+0300 (GTB Daylight Time)
ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '11-14-2019' for column 'belgeTarihi' at row 1
->Error: ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '11-14-2019' for column 'belgeTarihi' at row 1
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:107:40)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Tue Jun 18 2019 19:21:57 GMT+0300 (GTB Daylight Time)
ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '10-14-2019' for column 'belgeTarihi' at row 1
->Error: ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '10-14-2019' for column 'belgeTarihi' at row 1
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:107:40)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:462:44
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Tue Jun 18 2019 19:22:27 GMT+0300 (GTB Daylight Time)
ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '11-14-2019' for column 'belgeTarihi' at row 1
->Error: ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '11-14-2019' for column 'belgeTarihi' at row 1
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:107:40)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Tue Jun 18 2019 19:22:27 GMT+0300 (GTB Daylight Time)
ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '10-14-2019' for column 'belgeTarihi' at row 1
->Error: ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '10-14-2019' for column 'belgeTarihi' at row 1
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:107:40)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:462:44
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Tue Jun 18 2019 19:25:04 GMT+0300 (GTB Daylight Time)
ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '11-14-2019' for column 'belgeTarihi' at row 1
->Error: ER_TRUNCATED_WRONG_VALUE: Incorrect datetime value: '11-14-2019' for column 'belgeTarihi' at row 1
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:107:40)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Tue Jun 18 2019 19:25:05 GMT+0300 (GTB Daylight Time)
ER_BAD_FIELD_ERROR: Unknown column 'silindiMi' in 'where clause'
->Error: ER_BAD_FIELD_ERROR: Unknown column 'silindiMi' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.selectAll (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:54:21)
    at C:\Users\INTERSAN\Desktop\sbs\routes\index.js:411:39
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Wed Jun 19 2019 15:33:06 GMT+0300 (GTB Daylight Time)
veritabanihatasi
->undefined
Wed Jun 19 2019 15:33:06 GMT+0300 (GTB Daylight Time)
Connection lost: The server closed the connection.
->Error: Connection lost: The server closed the connection.
    at Protocol.end (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:112:13)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:97:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitNone (events.js:111:20)
    at Socket.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
Wed Jun 19 2019 23:33:06 GMT+0300 (GTB Daylight Time)
ER_BAD_FIELD_ERROR: Unknown column 'anketAdi' in 'field list'
->Error: ER_BAD_FIELD_ERROR: Unknown column 'anketAdi' in 'field list'
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:134:25)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:519:24
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Thu Jun 20 2019 12:35:30 GMT+0300 (GTB Daylight Time)
ER_BAD_FIELD_ERROR: Unknown column 'anketAdi' in 'field list'
->Error: ER_BAD_FIELD_ERROR: Unknown column 'anketAdi' in 'field list'
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:134:25)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:519:24
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Thu Jun 20 2019 13:04:01 GMT+0300 (GTB Daylight Time)
ER_NO_SUCH_TABLE: Table 'sbs_comp_2.ankentler' doesn't exist
->Error: ER_NO_SUCH_TABLE: Table 'sbs_comp_2.ankentler' doesn't exist
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:134:25)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:680:37
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Thu Jun 20 2019 13:44:44 GMT+0300 (GTB Daylight Time)
ER_BAD_FIELD_ERROR: Unknown column 'anketCinsi' in 'field list'
->Error: ER_BAD_FIELD_ERROR: Unknown column 'anketCinsi' in 'field list'
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.insert (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:134:25)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:680:37
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Thu Jun 20 2019 13:46:10 GMT+0300 (GTB Daylight Time)
ER_BAD_FIELD_ERROR: Unknown column 'anketId' in 'where clause'
->Error: ER_BAD_FIELD_ERROR: Unknown column 'anketId' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.selectQuery (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:68:21)
    at C:\Users\INTERSAN\Desktop\sbs\routes\index.js:419:41
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Thu Jun 20 2019 13:59:46 GMT+0300 (GTB Daylight Time)
veritabanihatasi
->undefined
Thu Jun 20 2019 13:59:46 GMT+0300 (GTB Daylight Time)
ER_BAD_FIELD_ERROR: Unknown column 'anketId' in 'where clause'
->Error: ER_BAD_FIELD_ERROR: Unknown column 'anketId' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.selectQuery (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:68:21)
    at C:\Users\INTERSAN\Desktop\sbs\routes\index.js:419:41
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Thu Jun 20 2019 14:00:48 GMT+0300 (GTB Daylight Time)
veritabanihatasi
->undefined
Thu Jun 20 2019 14:00:48 GMT+0300 (GTB Daylight Time)
ER_BAD_FIELD_ERROR: Unknown column 'ankedtId' in 'where clause'
->Error: ER_BAD_FIELD_ERROR: Unknown column 'ankedtId' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.setSilindi (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:210:21)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:668:24
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Thu Jun 20 2019 14:25:29 GMT+0300 (GTB Daylight Time)
ER_BAD_FIELD_ERROR: Unknown column 'anketAdi' in 'field list'
->Error: ER_BAD_FIELD_ERROR: Unknown column 'anketAdi' in 'field list'
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.selectWithColumn (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:86:21)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:851:27
    at Layer.handle [as handle_request] (C:\Users\INTERSAN\Desktop\sbs\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\INTERSAN\Desktop\sbs\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\INTERSAN\Desktop\sbs\node_modules\express\lib\router\route.js:112:3)
Thu Jun 20 2019 16:08:20 GMT+0300 (GTB Daylight Time)
ER_BAD_FIELD_ERROR: Unknown column 'anketAdi' in 'field list'
->Error: ER_BAD_FIELD_ERROR: Unknown column 'anketAdi' in 'field list'
    at Query.Sequence._packetToError (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:525:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\INTERSAN\Desktop\sbs\node_modules\mysql\lib\Connection.js:201:25)
    at Promise (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:12:29)
    at new Promise (<anonymous>)
    at Database.query (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:11:16)
    at Database.selectWithColumn (C:\Users\INTERSAN\Desktop\sbs\selfContent\database.js:86:21)
    at C:\Users\INTERSAN\Desktop\sbs\routes\ajax.js:851:27
    at Layer.handle [as handle_request] (C:\Users\INTERSAN\Desktop\sbs\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\INTERSAN\Desktop\sbs\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\INTERSAN\Desktop\sbs\node_modules\express\lib\router\route.js:112:3)
Thu Jun 20 2019 16:09:46 GMT+0300 (GTB Daylight Time)
(intermediate value).selectQuery(...).map is not a function
->TypeError: (intermediate value).selectQuery(...).map is not a function
    at C:\Users\INTERSAN\Desktop\sbs\routes\index.js:422:111
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Thu Jun 20 2019 17:25:03 GMT+0300 (GTB Daylight Time)
